migrations/1597882126530_create-collections.ts

```diff
@@ -0,0 +1,27 @@
+import { MigrationBuilder } from 'node-pg-migrate'
+import { Collection } from '../src/Collection'
+
+const tableName = Collection.tableName
+
+export const up = (pgm: MigrationBuilder) => {
+  pgm.createTable(
+    tableName,
+    {
+      id: { type: 'UUID', primaryKey: true, unique: true, notNull: true },
+      name: { type: 'TEXT', notNull: true },
+      eth_address: { type: 'TEXT', notNull: true },
+      hash: { type: 'UUID' },
+      contract_address: { type: 'TEXT' },
+      is_published: { type: 'TEXT', default: false, notNull: true },
```

Comment:
What do you think of using `BOOLEAN` here?

Reply:
It should be BOOLEAN, good catch